roles/install_dbserver/tasks/validate_install_dbserver.yml

```diff
@@ -99,7 +99,7 @@
 
 - name: Install python-apt package
   ansible.builtin.package:
-    name: python-apt
```

Comment:
I am worried about what would happen to the old version of Debian. Would you like to add a check for the Debian version and, based on that, use python3-apt?

Reply:
I could do that - do you think it's necessary? Debian removed python-apt five years ago, as far as I can tell, along with Python 2.7.